satellite/console/consoleweb/server.go

```diff
@@ -117,6 +117,20 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 
 // appHandler is web app http handler function
 func (s *Server) appHandler(w http.ResponseWriter, req *http.Request) {
+	header := w.Header()
+
+	cspValues := []string{
+		"default-src 'self'",
+		"script-src 'self' *.stripe.com cdn.segment.com",
+		"frame-src 'self' *.stripe.com",
+		"img-src 'self' data:",
+	}
+
+	header.Set("Content-Type", "text/html; charset=UTF-8")
+	header.Set("Content-Security-Policy", strings.Join(cspValues, "; "))
+
+	w.WriteHeader(200)
```

Comment:
i would prefer the actual html variable here, instead of the integer.

Reply:
I think, Go should automatically write it anyways, so need to explicitly add it.

Reply:
You are right